spec/support/request_specs.rb

```diff
@@ -1,11 +1,24 @@
 require_relative(File.expand_path('../../../app/service', __FILE__))
```

Comment:
One more thing: We don't need `require_relative` AND `File.expand_path`, right? That's redundant?